### PR TITLE
PE-7080: supply a handler for paginated delegates of a gateway

### PR DESCRIPTION
**Pull request by arielmelendez**

### CODE REVIEW

File: src/gar.lua [hunk 1]
```diff
@@ -875,6 +875,29 @@ function gar.getPaginatedGateways(cursor, limit, sortBy, sortOrder)
...
+		error("Gateway not found")
+	end
+	local delegatesArray = {}
+	local cursorField = "address"
+	for delegateAddress, delegate in pairs(gateway.delegates) do
+		delegate.address = delegateAddress
+		table.insert(delegatesArray, delegate)
+	end
+
+	print("number of delegates " .. utils.lengthOfTable(delegatesArray))
```
Comment (dtfiedler) [inline src/gar.lua:896]: nit: looks like testing log?

**Comment by dtfiedler** [comment]
need to update `handlers.test.mjs` to account for the new handler here: https://github.com/ar-io/ar-io-network-process/blob/develop/tests/handlers.test.mjs#L41

```
Subtest: handlers
    # Subtest: should maintain order of handlers, with _eval and _default first, followed by prune
    not ok 1 - should maintain order of handlers, with _eval and _default first, followed by prune
      ---
      duration_ms: 122.912957
      location: 'file:///home/runner/work/ar-io-network-process/ar-io-network-process/tests/handlers.test.mjs:24:3'
      failureType: 'testCodeFailure'
      error: 'should have 60 handlers; got 61'
      code: 'ERR_ASSERTION'
      name: 'AssertionError'
      expected: true
      actual: false
      operator: '=='
      stack: |-
        TestContext.<anonymous> (file:///home/runner/work/ar-io-network-process/ar-io-network-process/tests/handlers.test.mjs:45:12)
        async Test.run (node:internal/test_runner/test:632:9)
        async Promise.all (index 0)
        async Suite.run (node:internal/test_runner/test:948:7)
        async startSubtest (node:internal/test_runner/harness:216:3)
      ...
```

File: src/gar.lua (hunk 1 above)
Comment (arielmelendez) [reply src/gar.lua]: NOTHING TO SEE HERE! 3bb7fa2

**Comment by arielmelendez** [comment]
> need to update `handlers.test.mjs` to account for the new handler here: https://github.com/ar-io/ar-io-network-process/blob/develop/tests/handlers.test.mjs#L41
> 
> ```
> Subtest: handlers
>     # Subtest: should maintain order of handlers, with _eval and _default first, followed by prune
>     not ok 1 - should maintain order of handlers, with _eval and _default first, followed by prune
>       ---
>       duration_ms: 122.912957
>       location: 'file:///home/runner/work/ar-io-network-process/ar-io-network-process/tests/handlers.test.mjs:24:3'
>       failureType: 'testCodeFailure'
>       error: 'should have 60 handlers; got 61'
>       code: 'ERR_ASSERTION'
>       name: 'AssertionError'
>       expected: true
>       actual: false
>       operator: '=='
>       stack: |-
>         TestContext.<anonymous> (file:///home/runner/work/ar-io-network-process/ar-io-network-process/tests/handlers.test.mjs:45:12)
>         async Test.run (node:internal/test_runner/test:632:9)
>         async Promise.all (index 0)
>         async Suite.run (node:internal/test_runner/test:948:7)
>         async startSubtest (node:internal/test_runner/harness:216:3)
>       ...
> ```

57083b7 There will be another as I add the allowed delegates handler.

**Review by dtfiedler — APPROVED** [review]

**Pull request merged**